Silent-Vampires/Welcome-to-Nashville-Silent-Vampires
# Rc new file structure

# Description

Completely changed file structure to what we discussed. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout rc-new-file-structure
run http server
test meetup button to confirm that it prints to the DOM



# Checklist:
